esp-box/include/bsp/esp-box.h

```diff
@@ -162,17 +149,19 @@ extern "C" {
  *      - ESP_ERR_NO_MEM        No memory for storing the channel information
  *      - ESP_ERR_INVALID_STATE This channel has not initialized or already started
  */
-esp_err_t bsp_audio_init(const i2s_std_config_t *i2s_config, i2s_chan_handle_t *tx_channel, i2s_chan_handle_t *rx_channel);
+#if ESP_IDF_VERSION < ESP_IDF_VERSION_VAL(5, 0, 0)
+esp_err_t bsp_audio_init(const i2s_config_t *i2s_config);
+#else
+esp_err_t bsp_audio_init(const i2s_std_config_t *i2s_config);
+#endif
 
 /**
- * @brief Enable/disable audio power amplifier
+ * @brief Get codec I2S interface (initialized in bsp_audio_init)
  *
- * @param[in] enable Enable/disable audio power amplifier
  * @return
- *      - ESP_OK                On success
- *      - ESP_ERR_INVALID_ARG   Invalid GPIO number
+ *      - Pointer to codec I2S interface handle or NULL when error occured
  */
-esp_err_t bsp_audio_poweramp_enable(const bool enable);
```

Comment:
Is it a breaking change to remove this function?

Reply:
Yes. The whole audio changes to `esp_codec_dev` are breaking changes.

Reply:
In that case, why is the major version of the component not incremented? https://semver.org/#summary